src/greedy.jl

```diff
@@ -106,12 +129,10 @@ function assemble(info::NamedTuple, H::AffineDecomposition, grid, greedy::Greedy
         end
 
         # Construct initial guess at μ_next and run truth solve
-        if greedy.init_from_rb
-            Ψ₀ = interpolate(info.basis, info.h_cache.h, μ_next, solver_truth, solver_online)
-        else
-            Ψ₀ = nothing
-        end
-        truth = solve(H, μ_next, Ψ₀, solver_truth)
+        Ψ_init = greedy.Ψ_init((; info..., err_grid, λ_grid, err_max, μ=μ_next),
+                               (; H, grid, greedy, solver_truth, compressalg,
+                                solver_online))
```

Comment:
Why does this need to be explicitly created? This looks a bit odd.

Reply:
I wanted the initial guess function to have access to all `assemble(...)` arguments to keep the interface general, e.g. the RB initial guesses need to have access to the `solver_online` variable. The initial guess functions now have the signature `guess(info, offline_args)` which makes picking out specific arguments easy. I could also directly pass all offline arguments and then have `guess(info, offline_args...)` —  do you think this would make more sense?

Reply:
Hmm. Not sure I was more thinking it might be nicer to have all quantities in this named tuple be assigned to a variable rather than always constructing it in place. But if it's only used once let's keep it.